examples/pylab_examples/layer_images.py

```diff
@@ -23,13 +23,10 @@ def func3(x, y):
 # for the images their apparent extent could be different due to
 # interpolation edge effects
 
-
-xmin, xmax, ymin, ymax = np.amin(x), np.amax(x), np.amin(y), np.amax(y)
-extent = xmin, xmax, ymin, ymax
+extent = np.min(x), np.max(x), np.min(y), np.max(y)
 fig = plt.figure(frameon=False)
 
-Z1 = np.array(([0, 1]*4 + [1, 0]*4)*4)
-Z1.shape = (8, 8)  # chessboard
+Z1 = np.add.outer(range(8), range(8)) % 2  # chessboard
```

Comment:
It makes the first argument a column array and the second a row array, broadcasts them and applies the ufunc.